docs/pushnotificationios.md

```diff
@@ -116,24 +112,23 @@ Then enable Background Modes/Remote notifications to be able to use remote notif
 static presentLocalNotification(details: PresentLocalNotificationDetails);
 ```
 
-Schedules the localNotification for immediate presentation.
+Schedules a local notification for immediate presentation.
 
 **Parameters:**
 
 | Name    | Type   | Required | Description |
 | ------- | ------ | -------- | ----------- |
 | details | object | Yes      | See below.  |
 
-details is an object containing:
+`details` is an object containing:
 
-- `alertBody` : The message displayed in the notification alert.
-- `alertAction` : The "action" displayed beneath an actionable notification. Defaults to "view". Note that Apple no longer shows this in iOS 10 +
```

Comment:
Should we still call out the fact that `alertAction` still exists but is deprecated?